game.rb

```diff
@@ -1,15 +1,47 @@
 require './map.rb'
+require './player.rb'
 
 class Game
 
+  def initialize
+    map = Map.new
+    player1 = Player.new "O"
+    player2 = Player.new "X"
+    map.initialize_empty
+    map.draw_map(map.get_array_map)
+  end
+
   def get_position(sign)
     puts "Podaj pole w którym mam wpisać #{sign}"
     position = gets.chomp
     print position
   end
 
+  def tour(player)
+
+  end
 end
 
+game = Game.new
```

Comment:
nie jestem fanem tak wielu linijek kodu tutaj. Jednak powinno być to zawarte w jakiejś osobnej klasie lub metodzie, a potem wywoływana tylko ona.

Reply:
Błąd w aplikacji wynika z błędnego argumentu przekazanego do funkcji. W pliku game.rb w linii 36 powinno być: map.check_draw(map.get_array_map)